elements/lisk-cryptography/test/bls_lib/lib.spec.ts

```diff
@@ -60,231 +59,259 @@ interface EthFastAggrVerifySpec {
 }
 
 describe('bls_lib', () => {
-	const groupOrder = '0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001';
+	describe('isNotMultipleOfGroupOrder', () => {
+		it('should return true if sk is not equal to groupOrder or groupOrderDouble', () => {
+			const sk = Buffer.from(
+				'73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000003',
+				'hex',
+			);
+			expect(isNotMultipleOfGroupOrder(sk)).toBe(true);
+		});
 
-	describe('isSecretKeyNonZeroModOrder', () => {
-		it('should return true when given a valid secret key', () => {
-			const secretKey = SecretKey.fromBytes(Buffer.alloc(32, 1));
-			expect(isSecretKeyNonZeroModOrder(secretKey)).toBe(true);
+		it('should return true if sk is 3*groupOrder', () => {
```

Comment:
I would remove this test because it is not a requirement that `isNotMultipleOfGroupOrder` returns `true` in this case. In fact, it wouldn't hurt if we extend the function to also compare against `3*r`, `4*r`,... but we just don't do it for efficiency.